chatbot/services/response_matcher.py

```
@@ -0,0 +1,241 @@
+        Parameters:
+            pattern (str): الگوی جستجو؛ می‌تواند یک رشته ساده یا یک عبارت‌نگارهٔ regex باشد.
+            message (str): متن پیام که در آن جستجو انجام می‌شود.
+        
+        Returns:
+            bool: True اگر الگو در پیام مطابق باشد، در غیر این صورت False.
+        """
+        try:
+            # بررسی اینکه آیا pattern یک regex است
+            if any(char in pattern for char in r'.*+?[]{}()|^$\\'):
+                return bool(re.search(pattern, message))
+            else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)